test-VM.nix

```
@@ -2,7 +2,10 @@
   pkgs ? import (fetchTarball {
     url = "https://github.com/NixOS/nixpkgs/archive/refs/tags/22.11.tar.gz";
     sha256 = "sha256:11w3wn2yjhaa5pv20gbfbirvjq6i3m7pqrq2msf0g7cv44vijwgw";
-  }) {}
+  }) {},
+  platform ? " ",
+  tests ? " ",
+  suites ? " "
```

Standardize (compliant with terms in baremetal-tf.nix)
```suggestion
  list_tests ? " "
```